File: src/ERP/acme.atena.domain/DTO/Account/FormaPagamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.atena.domain.DTO.Account
{
    public class FormaPagamento: AbstractEntity
    {
        public string Nome { get; set; }
        public int Codigo { get; set; }

        public virtual ICollection<PagamentoFormaPagamento> PagamentoFormaPagamentos { get; set; } = new HashSet<PagamentoFormaPagamento>();
        public virtual ICollection<PagamentoVendaFormaPagamento> PagamentoVendasFormasPagamentos { get; set; } = new HashSet<PagamentoVendaFormaPagamento>();

    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/account/forma-pagamento.ts
import { PagamentoVendaFormaPagamento } from './pagamento-venda-forma-pagamento';
import { AbstractEntity } from "../abstract-entity";
import { PagamentoFormaPagamento } from "./pagamento-forma-pagamento";

export class FormaPagamento extends AbstractEntity
{
  public  nome :string;
  public  codigo : number;

  public pagamentoFormaPagamentos : PagamentoFormaPagamento[];
  public pagamentoVendasFormasPagamentos: PagamentoVendaFormaPagamento[];

}
